docs-ref-services/storage.md

```diff
@@ -16,19 +15,79 @@ The Azure Storage libraries for Java provide classes for working with data in yo
+
+## Available packages
+
+ The following table describes the recommended versions of the storage client library for Java.
 
 | Library version | Supported services | Maven | Reference | Source, Readme, Examples |
 |----------------------|------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|-----------------------------------------------------------------------------------------------------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
-| Version 12 (preview) | Blob, Queue, and File | [Blob](https://search.maven.org/artifact/com.azure/azure-storage-blob/)<br />[Queue](https://search.maven.org/artifact/com.azure/azure-storage-queue/)<br />[File](https://search.maven.org/artifact/com.azure/azure-storage-file/) | [Version 12 reference](https://docs.microsoft.com/java/api/overview/azure/storage?view=azure-java-preview) | [Blob](https://github.com/Azure/azure-sdk-for-java/tree/master/sdk/storage/azure-storage-blob)<br />[Queue](https://github.com/Azure/azure-sdk-for-java/tree/master/sdk/storage/azure-storage-queue)<br />[File](https://github.com/Azure/azure-sdk-for-java/tree/master/sdk/storage/azure-storage-file) |
-| Version 8 | Blob, Queue, File, and Table | [All services](https://mvnrepository.com/artifact/com.microsoft.azure/azure-storage) | [Version 8 reference](https://docs.microsoft.com/java/api/overview/azure/storage/client?view=azure-java-stable) | [All services](https://github.com/azure/azure-storage-java/tree/legacy-master) |
+| Version 12 | Blob and Queue | [Blob](https://search.maven.org/artifact/com.azure/azure-storage-blob/)<br />[Queue](https://search.maven.org/artifact/com.azure/azure-storage-queue/) | [Blob](https://azuresdkdocs.blob.core.windows.net/$web/java/azure-storage-blob/12.0.0/index.html)<br/>[Queue](https://azuresdkdocs.blob.core.windows.net/$web/java/azure-storage-queue/12.0.0/index.html) | [Blob](https://github.com/Azure/azure-sdk-for-java/tree/azure-storage-blob_12.0.0/sdk/storage/azure-storage-blob)<br />[Queue](https://github.com/Azure/azure-sdk-for-java/tree/azure-storage-blob_12.0.0/sdk/storage/azure-storage-queue) |
+| Version 8 | Blob, Queue, File, and Table | [All services](https://mvnrepository.com/artifact/com.microsoft.azure/azure-storage) | [Version 8 reference](https://docs.microsoft.com/java/api/overview/azure/storage/client?view=azure-java-stable)<br/>[Quickstart](/azure/storage/blobs/storage-quickstart-blobs-java) | [All services](https://github.com/azure/azure-storage-java/tree/legacy-master) |
```

Comment:
Quickstart link doesn't work

Reply:
Link is relative to docs, so it should work once the build is done.  I didn't change it - just moved it.

Reply:
Okay - but that link points to v12 not v8 :). v8 would use /azure/storage/blobs/storage-quickstart-blobs-java-legacy